.rubocop.yml

```diff
@@ -34,32 +35,40 @@ Style/EachWithObject:
   Enabled: false
 Style/FrozenStringLiteralComment:
   Enabled: false
+Style/StringConcatenation:
+  Enabled: false
+Style/RedundantAssignment:
+  Enabled: false
 
 Naming/HeredocDelimiterNaming:
   Enabled: false
-Naming/UncommunicativeMethodParamName:
+Naming/MethodParameterName:
   Enabled: false
 Naming/ConstantName:
   Enabled: false
 Naming/AccessorMethodName:
   Enabled: false
 Naming/MethodName:
   Enabled: false
+Naming/FileName:
+  Enabled: false
```

Comment:
This rule suggest to rename `lib/line-bot-sdk.rb` to CamelCase. We can't accept the suggestion.